From 9926c32f5520abfb38b4223b3db7b89e7d51b709 Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Thu, 10 Jul 2014 01:07:34 -0400
Subject: [acpi] Remove acpi_pci_slot_init() headers from internal.h

Message-id: <20140710010734.22146.73677.stgit@gir.stowe>
Patchwork-id: 85629
O-Subject: [RHEL7.1 PATCH 090/105] ACPI: Remove acpi_pci_slot_init() headers from internal.h
Bugzilla: 1114228
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1114228
Upstream Status: 71bba8fafac8975dbb684df4098d2dd6baac1fda

commit 71bba8fafac8975dbb684df4098d2dd6baac1fda
Author: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
Date:   Thu Nov 7 01:40:50 2013 +0100

    ACPI: Remove acpi_pci_slot_init() headers from internal.h

    Since acpi_pci_slot_init() is now called from acpi_pci_init()
    and pci-acpi.h contains its header, remove that header (and the
    empty definition of that function for CONFIG_ACPI_PCI_SLOT unset)
    from internal.h as it doesn't have to be there any more.  That also
    avoids a build warning about duplicate function definitions for
    CONFIG_ACPI_PCI_SLOT unset.

    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/acpi/internal.h b/drivers/acpi/internal.h
index 3a50a34..016a4f8 100644
--- a/drivers/acpi/internal.h
+++ b/drivers/acpi/internal.h
@@ -25,11 +25,6 @@
 
 int init_acpi_device_notify(void);
 int acpi_scan_init(void);
-#ifdef CONFIG_ACPI_PCI_SLOT
-void acpi_pci_slot_init(void);
-#else
-static inline void acpi_pci_slot_init(void) { }
-#endif
 void acpi_pci_root_init(void);
 void acpi_pci_link_init(void);
 void acpi_pci_root_hp_init(void);
-- 
1.7.1